pkg/tracing/otel_trace_provider.go

```diff
@@ -0,0 +1,76 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/justtrackio/gosoline/pkg/appctx"
+	"github.com/justtrackio/gosoline/pkg/cfg"
+	"github.com/justtrackio/gosoline/pkg/log"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type OtelSettings struct {
+	Exporter      string  `cfg:"exporter"`
```

Comment:
Does it make sense to provide the name of the default exporter as default as well?

Reply:
added a default value